### PR TITLE
Move Settings page out of user dropdown

### PR DESCRIPTION
**Pull request by siggisim**
Feels more discoverable, but open to ideas:
Screen Shot 2022-10-05 at 1 51 37 PM

---

**Version bump**: Patch

### CODE REVIEW

File: enterprise/app/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -105,6 +105,10 @@ export default class SidebarComponent extends React.Component<Props, State> {
     return this.props.path.startsWith("/docs/setup/");
   }
 
+  isSettingsSelected() {
+    return this.props.path.startsWith("/settings");
```
Comment (bduffany) [inline enterprise/app/sidebar/sidebar.tsx:109]:
```suggestion
    return this.props.path.startsWith("/settings/");
```

**Review by bduffany — APPROVED** [review]
"Setup" next to "Settings" feels a bit stuttery, maybe at some point we could change "Setup" to "Quickstart" or something similar?

**Comment by siggisim** [comment]
> "Setup" next to "Settings" feels a bit stuttery / ambiguous, maybe at some point we could change "Setup" to "Quickstart" or something similar?

Done, I think this looks nice - I updated references to it in the docs too.

Screen Shot 2022-10-05 at 3 21 50 PM

File: enterprise/app/sidebar/sidebar.tsx (hunk 1 above)
Comment (siggisim) [reply enterprise/app/sidebar/sidebar.tsx]: Done

**Comment by bduffany** [comment]
> Done, I think this looks nice - I updated references to it in the docs too.

Nice! I love the choice of icon

**Pull request merged**